Automattic/wp-calypso
# Design updates for the DIFM flow

#### Changes proposed in this Pull Request
Context: pdh1Xd-N7-p2#comment-1039

* ac32d633e8 Updates subheader of social profiles step
* f76ee38c79 Fixes styling for the social profiles step
* 9b52087ab6 Updates subheader copy of page picker step
* 2dd3479a1a Replace podcast with menu page
* 4b7cffbdcf Replace Showcase A Service with Services
* 2b2dfb333f Adds required badge to home page cell
* 88dc8b266c Updates subheader of website content step


#### Testing instructions



* Go to `/start/do-it-for-me?flags=signup/redesigned-difm-flow`
* Click on "New Site" and then after entering the site name, proceed to the social profiles step.
* Confirm that the subheader on this step is 'You can always add them later.'
<img width="559" alt="image" src="https://user-images.githubusercontent.com/5436027/161914661-c7f30103-82a5-4b22-937c-12b190757b95.png">

* Confirm that the styling for the text inputs matches with the one on the previous step (site title step). Check the hover and focus behaviour.

https://user-images.githubusercontent.com/5436027/161917517-2e631b97-8a1c-4e36-a0d8-1bb8a5b68828.mp4

* Proceed to the design picker step, select a design and proceed to the page picker step.
* Confirm that the subheader copy has been updated.
<img width="502" alt="image" src="https://user-images.githubusercontent.com/5436027/161917947-bf53f7b2-6d87-4297-845c-698f9a3dfc35.png">

* Confirm that the Podcast and Showcase A Service page options have been replaced by Menu and Services respectively.
<img width="871" alt="image" src="https://user-images.githubusercontent.com/5436027/161918272-c91be2ff-3f0d-430f-89d3-160cf69a8b30.png">

* Confirm that the "Home" page cell has a "Required" badge instead of a "Popular" badge.
<img width="258" alt="image" src="https://user-images.githubusercontent.com/5436027/161918946-5a235356-9952-48eb-8e16-525138cc60ee.png">

* Proceed to checkout. After checkout, on the website content step, confirm that the subheader has been updated:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/5436027/161919265-c2879e13-cafe-413c-a4e2-856292b98238.png">



Related to 0-as-1202078586852352/f
